tasks/custom_task/custom_task.py

```diff
@@ -44,10 +44,10 @@ def get_running_modes() -> list[str]:
     # When running the unit tests with the invoke command, the INVOKE_UNIT_TESTS env variable is set.
     is_running_ut = "unittest" in " ".join(sys.argv)
     running_modes = {
-        "pre_commit": os.environ.get("PRE_COMMIT", 0) == "1",
+        "pre_commit": running_in_pre_commit(),
         "invoke_unit_tests": is_running_ut or os.environ.get("INVOKE_UNIT_TESTS", 0) == "1",
```

Comment:
I think that it would be good to do the same for `INVOKE_UNIT_TESTS`

Reply:
I'll let @amenasria find the best approach for this one since this is not working exactly the same way